qemu/tests/cfg/block_hotplug.cfg

```diff
@@ -11,10 +11,17 @@
     remove_image_stg0 = yes
     force_create_image_stg0 = yes
     get_disk_cmd = "ls /dev/[hsv]d[a-z]* | sort"
-    pci_test_cmd = "dd if=%s of=/dev/null bs=1k count=1000 iflag=direct &&"
-    pci_test_cmd += " dd if=/dev/zero of=%s bs=1k count=1000 oflag=direct"
+    disk_op_cmd = "dd if=%s of=/dev/null bs=1k count=1000 iflag=direct &&"
+    disk_op_cmd += " dd if=/dev/zero of=%s bs=1k count=1000 oflag=direct"
     kill_vm_on_error = yes
 
+    windows:
+        disk_index = "1 2"
+        disk_letter = "I J"
+        format_disk = yes
+        iozone_cmd = "D:\Iozone\iozone.exe -azR -r 64k -n 125M -g 512M -M -i 0"
```

Comment:
If here the `D` means you are using `WINUTILS`, please get it dynamically by calling `utils_misc.get_winutils_vol`.

Reply:
thanks, has changed~